OWML.Common/Menus/IModConfigMenuBase.cs

```diff
@@ -5,6 +5,7 @@ public interface IModConfigMenuBase : IModPopupMenu
         IModManifest Manifest { get; }
 
         void Initialize(Menu modMenuCopy, IModToggleInput toggleTemplate, IModSliderInput sliderTemplate, 
-            IModTextInput textInputTemplate, IModNumberInput numberInputTemplate, IModComboInput comboInputTemplate);
+            IModTextInput textInputTemplate, IModNumberInput numberInputTemplate,
+            IModComboInput comboInputTemplate, IModSelectorInput selectorTemplate);
```

Comment:
If we get to a point where we need to have line breaks between method arguments, we should always have only one argument per line, or it becomes awkward to read (especially in diffs).

I know this example was already split before, but it's a good chance to clean it up.

I think @amazingalek agreed with me before on this, but maybe that was on the QSB codebase? What do you think.

Reply:
IMO it's kinda out of scope of this PR, possibly an extra code cleanup round by Alek/Me/You will do that 🤷‍♂️

Reply:
Boy scout rule again :) Small clean up (not big refactors) is ok when you're touching messy code.